src/scripts/syncBreaches.js

```diff
@@ -53,36 +53,43 @@ async function uploadToS3(fileName, fileStream) {
 }
 
 export async function getBreachIcons(breaches) {
-  const breachDomains = breaches
-    .map((breach) => breach.Domain)
-    .filter((breachDomain) => breachDomain.length > 0);
+  const filteredBreaches = breaches.filter(async ({Domain, Name}) => {
+    const domainExists = Domain.length > 0
+    // if domain does not exist, null the logo path
+    if (!domainExists) await updateBreachLogoPath(Name, null)
+    return domainExists
+  });
 
   // make logofolder if it doesn't exist
   const logoFolder = os.tmpdir();
   console.log(`Logo folder: ${logoFolder}`)
 
   // read existing logos
   const existingLogos = await readdir(logoFolder);
-  console.log(`existing logos: ${existingLogos.length}`)
 
   (await Promise.all(
-    breachDomains.map(async (breachDomain) => {
+    filteredBreaches.map(async ({Domain: breachDomain, Name: breachName}) => {
       const logoFilename = breachDomain.toLowerCase() + ".ico";
       const logoPath = pathResolve(logoFolder, logoFilename);
       if (existingLogos.includes(logoFilename)) {
         console.log('skipping ', logoFilename)
+        await updateBreachLogoPath(breachName, `https://s3.amazonaws.com/${process.env.S3_BUCKET}/${breachDomain.toLowerCase()}.ico`)
```

Comment:
Nit, channelling my inner Peter :smile::

```suggestion
        await updateBreachLogoPath(breachName, `https://s3.amazonaws.com/${process.env.S3_BUCKET}/${logoFilename}`)
```